c/certificateService.c

```diff
@@ -210,9 +254,9 @@ static int serveMappingService(HttpService *service, HttpResponse *response)
   return 0;
 }
 
-void installCertificateService(HttpServer *server)
+void installUserMappingService(HttpServer *server)
 {
-  HttpService *httpService = makeGeneratedService("CertificateService", "/certificate/x509/**");
+  HttpService *httpService = makeGeneratedService("UserMappingService", "/user-mapping/**");
```

Comment:
This looks like a breaking change - you can't rename URLs like this without a v3 because someone might be using it.

Reply:
this is something I would like to discuss. Is it better to keep the misleading URL or rather do this change if it is probable that API ML is the only user of this service.